tests/attr/test_shapley.py

```diff
@@ -39,6 +40,29 @@ def test_simple_shapley_sampling_with_mask(self) -> None:
             perturbations_per_eval=(1, 2, 3),
         )
 
+    def test_simple_shapley_sampling_boolean(self) -> None:
+        net = BasicModelBoolInput()
+        inp = torch.tensor([[True, False, True]])
+        self._shapley_test_assert(
+            net,
+            inp,
+            [35.0, 35.0, 35.0],
+            feature_mask=torch.tensor([[0, 0, 1]]),
+            perturbations_per_eval=(1, 2, 3),
+        )
+
+    def test_simple_shapley_sampling_boolean_with_baseline(self) -> None:
+        net = BasicModelBoolInput()
```

Comment:
Another way of testing would be using `BasicModel_MultiLayer()` with the transformed input and seeing if we get the same result but if we manually verified expected value, then it looks good to me.

Reply:
Yes, I double checked the values manually, they seem correct.